src/htmlize/main.ml

```diff
@@ -52,6 +52,11 @@ let htmlize filename content =
     Printf.bprintf b {|%s|} content;
     Buffer.contents b
 
+  | "jpeg" | "jpg" | "png" | "gif" | "tiff" | "raw" ->
```

Comment:
JPG JPEG svg ico

Reply:
Here's [a list](https://developer.mozilla.org/fr/docs/Web/Media/Formats/Image_types) :)

Reply:
That has already been done at line 48 !